packages/core/app-extensions/src/actions/components/SingleAction.js

```diff
@@ -4,29 +4,31 @@ import {Button} from 'tocco-ui'
 
 import {isValidSelection, selectionText} from './selectionHelper'
 
-export const SingleAction = ({definition, onClick, selectedCount, disabled, intl}) => (
-  <Button
-    iconOnly={definition.buttonType === 'ICON'}
-    data-cy={`action-${definition.id}`}
-    look={definition.buttonType === 'ICON' ? 'flat' : 'raised'}
-    withoutBackground={definition.buttonType === 'ICON'}
-    onClick={e => {
-      onClick(definition)
-      e.stopPropagation()
-    }}
-    icon={definition.icon}
-    label={getLabel(definition)}
-    title={getTitle(selectedCount, definition, intl)}
-    disabled={definition.readonly === true || !isValidSelection(selectedCount, definition) || disabled}
-  />
-)
+export const SingleAction = ({definition, onClick, selectedCount, disabled, intl}) => {
+  const shouldUseLabel = def => !(def.useLabel === 'NO' || def.useLabel === 'HIDE')
+  const isDisabled = definition.readonly === true || !isValidSelection(selectedCount, definition) || disabled
+  const getLabel = definition.label && shouldUseLabel(definition) ? definition.label : ''
```

Comment:
nitpick: I personally would prefer another name here

`getLabel` sounds like a function but instead it's a variable.
I would prefer `label`